### PR TITLE
Turret Gun update

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by theOperand** [comment]
1000/10 merge posthaste

i can see the amount of thought and balance that went into this, this new feature makes my eyes tear. the code is perfect, and I do not see how anyone could dislike this. You should be made headcoders of all of SS13 right now.

**Comment by fleure** [comment]
Thanks you too

(Did you guys mean to create a clone of NTStation to code for?)

File: code/game/machinery/jukebox.dm [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	file = 'sound/voice/TestLoop1.ogg'
+	falloff = 2
+	repeat = 1
+
+/mob/var/music = 0
+
+var/getmusic = ""
+var/songsText = ""
+//online list:
+
```
Comment (theOperand) [inline code/game/machinery/jukebox.dm:18]: lol, this is literally stolen d2k5 code, i remember looking at this and laughing.

File: code/modules/powerarmor/components/seals.dm [hunk 2]
```diff
@@ -160,7 +160,7 @@
 				parent.helm = null
 				helm.parent = null
 
-/obj/item/weapon/powerarmor/atmoseal/optional/standard // Spaceproof, but not fireproof
+/obj/item/weapon/powerarmor/atmoseal/optional/standart // Spaceproof, but not fireproof
```
Comment (theOperand) [inline code/modules/powerarmor/components/seals.dm:163]: lol

File: code/modules/powerarmor/components/weapons.dm [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
-// melee
-/obj/item/weapon/powerarmor/weapon/melee/multiplier
+// MEELE
+/obj/item/weapon/powerarmor/weapon/meele/multiplier
 	name = "punch repeater"
 	desc = "Repeats your hand's movements with a set of servos."
 	origin_tech = "combat=2;materials=1;programming=1;engineering=1"
 
-/obj/item/weapon/powerarmor/weapon/melee/multiplier/pattack(var/atom/A, var/mob/living/carbon/user)
+/obj/item/weapon/powerarmor/weapon/meele/multiplier/pattack(var/atom/A, var/mob/living/carbon/user)
```
Comment (theOperand) [inline code/modules/powerarmor/components/weapons.dm:11]: LOL

**Comment by Kraseo** [comment]
I give it a 10/Я

File: code/game/machinery/jukebox.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/machinery/jukebox.dm:18]: vg fixed it up and made it work better, this is probably the vg fixup

besides wasn't it technically released as opensource anyways? nernums and that other vg guy released it

Comment (theOperand) [inline code/game/machinery/jukebox.dm:18]: its a leak
and its not clenaed up, it still has the spawn(rand(3,10)) which is just hilarious

File: code/modules/food/recipes_fryer.dm [hunk 4]
```diff
@@ -0,0 +1,46 @@
+
+// see code/datums/recipe.dm
+
+/datum/recipe/fryer/cheesyfries
```
Comment (Iamgoofball) [inline code/modules/food/recipes_fryer.dm:4]: THIS IS NOT THE POINT OF THE DEEP FRYER

YOU FUCKING RUINED IT

File: code/modules/food/customizables.dm [hunk 5]
```diff
@@ -3,20 +3,145 @@
 		var/obj/item/weapon/reagent_containers/food/snacks/customizable/S = new(get_turf(user))
 		S.attackby(W,user)
 		qdel(src)
-/obj/item/weapon/reagent_containers/food/snacks/bun/attackby(obj/item/W as obj, mob/user as mob)
-	if(istype(W,/obj/item/weapon/reagent_containers/food/snacks))
+
+/obj/item/weapon/reagent_containers/food/snacks/bun/attackby(obj/item/W as obj, mob/user as mob) //Looks like shit, but whatever -- Smet19
```
Comment (Iamgoofball) [inline code/modules/food/customizables.dm:7]: THIS IS NOT THE POINT OF THE CUSTOMIZABLE FOODS AT ALL

YOU FUCKING RUINED IT

File: code/modules/food/recipes_grill.dm [hunk 6]
```diff
@@ -0,0 +1,196 @@
+
+// see code/datums/recipe.dm
+
+/datum/recipe/grill/friedegg
```
Comment (Iamgoofball) [inline code/modules/food/recipes_grill.dm:4]: THIS IS NOT THE POINT OF THE GRILL

YOU FUCKING RUINED IT

File: code/modules/food/recipes_oven.dm [hunk 7]
```diff
@@ -0,0 +1,675 @@
+
+// see code/datums/recipe.dm
+
+/datum/recipe/oven/jellydonut
```
Comment (Iamgoofball) [inline code/modules/food/recipes_oven.dm:4]: THIS IS NOT THE POINT OF THE OVEN

YOU FUCKING RUINED IT

File: code/game/machinery/kitchen/deep_fryer.dm [hunk 8]
```diff
@@ -8,55 +8,189 @@
 	anchored = 1
 	use_power = 1
 	idle_power_usage = 5
+	flags = OPENCONTAINER | NOREACT
 	var/on = FALSE	//Is it deep frying already?
 	var/obj/item/frying = null	//What's being fried RIGHT NOW?
+	var/global/list/datum/recipe/fryer/available_recipes // List of the recipes you can use //Coderbus will kill me
```
Comment (Iamgoofball) [inline code/game/machinery/kitchen/deep_fryer.dm:14]: NO

THIS IS NOT THE POINT OF THE FRYER

YOU FUCKING RUINED IT

THIS IS A FUCKING CRAPPY ASS COPYPASTA OF THE MICROWAVE

REVERT THIS SHIT

File: code/game/machinery/kitchen/food_grill.dm [hunk 9]
```diff
@@ -8,47 +8,181 @@
 	anchored = 1
 	use_power = 1
 	idle_power_usage = 5
+	flags = OPENCONTAINER | NOREACT
 	var/on = FALSE	//Is it grilling food already?
+	var/global/list/datum/recipe/grill/available_recipes // List of the recipes you can use //Coderbus will kill me
```
Comment (Iamgoofball) [inline code/game/machinery/kitchen/food_grill.dm:13]: SAME HERE, THIS IS FUCKING TERRIBLE AND YOU'VE RUINED IT

**Comment by GunHog** [comment]
Здравствуйте, русские друзья! Вы отправляете к неправильному коду! Были ли вы вместо попытке загрузить наш код? Если это так, пожалуйста, раскошелиться его, прежде чем продолжить!

**Comment by smet19** [comment]
Fucken lold
2Iamgoofball  I KNOW IT, IM WORKING ON IT

**Comment by v-starodubov** [comment]
GunHog, sorry for this guy.

**Comment by MishaPepyaka** [comment]
GunHog, this automatic translation is unreadable, we can read english text as well as russian.
MyLittleLoly, what a lovely commit conversation you have here, dude.

**Comment by v-starodubov** [comment]
MishaPepyaka, /rus/ development

**Comment by theOperand** [comment]
> borked merges

;_;

File: code/game/machinery/jukebox.dm (hunk 1 above)
Comment (v-starodubov) [inline code/game/machinery/jukebox.dm:18]: we dont use this code, just a mess

Comment (theOperand) [inline code/game/machinery/jukebox.dm:18]: Then why is it here ;_;

**Comment by ZLOFENIX** [comment]
Wow such code many commits so conversation
Чтож вы делаете, свидомиты?
http://oi39.tinypic.com/sevpz8.jpg

**Comment by MishaPepyaka** [comment]
ZLOFENIX, Прославляют наше комьюнити.

File: code/game/machinery/jukebox.dm (hunk 1 above)
Comment (smet19) [inline code/game/machinery/jukebox.dm:18]: Because we are lazzy assholes

**Comment by ColonicAcid** [comment]
Oh my! My own little doter 2!

P-please let me keep them, I promise I'll keep them in line!

**Comment by GunHog** [comment]
So....has anyone bothered to tell you guys that you are trying to submit a pull request to the wrong repository? I am sure it is obvious, but then you have not closed it yet...

**Comment by ColonicAcid** [comment]
@GunHog Don't you try and take my qt 3.14s away from me you monster.

**Comment by v-starodubov** [comment]
@HunterMaksim, close dat shit

**Comment by ghost** [comment]
Epic Fail, so EPIC

**Comment by iamgoofballsecondfork** [comment]
and so we never made contact with the Russian SS13 community again

**Comment by v-starodubov** [comment]
@iamgoofballsecondfork, Sanctions!

**Comment by Zedliser** [comment]
Let's use ukrainian code instead of russian!

**Comment by ColonicAcid** [comment]
Eh I mean Ukranian coders did manage to make the Metro series while they were stuck in their shed because >too poor to buy actual good negative temperature locks.
It's better than nothing.

All you russians are still my waifus tho!

**Comment by volas** [comment]
Снимаю шляпу. Этот лог коммитов - маленький шедевр.

**Comment by bear1ake** [comment]
Жесть...

**Pull request closed without merging**